_posts/en/newsletters/2025-03-21-newsletter.md

```diff
@@ -0,0 +1,223 @@
+---
+title: 'Bitcoin Optech Newsletter #346'
+permalink: /en/newsletters/2025/03/21/
+name: 2025-03-21-newsletter
+slug: 2025-03-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a discussion about LND's updated
+dynamic feerate adjustment system.  Also included are our regular
+sections describing recent changes to services, client software, and
+popular Bitcoin infrastructure software.
+
+## News
+
+- **Discussion of LND's dynamic feerate adjustment system:** Matt
+  Morehouse [posted][morehouse sweep] to Delving Bitcoin a description
+  of LND's recently-rewritten _sweeper_ system, which
+  determines the feerates to use for onchain transactions (including
+  [RBF][topic rbf] fee bumps).  He begins with a brief description
+  of the safety critical aspects of fee management for an LN node, as
+  well as the natural desire to avoid overpaying.  He then describes the
+  two general strategies used by LND:
+
+  - Querying external feerate estimators, such as a local Bitcoin Core
+    node or a third party.  This is mainly used for choosing initial
+    feerates and fee bumping non-urgent transactions.
+
+  - Exponential fee bumping, used when a deadline is approaching to
+    ensure that problems with a node's mempool or its [fee estimation][topic fee estimation]
+    don't prevent timely confirmation.  For example, Eclair uses exponential fee
+    bumping when deadlines are within six blocks.
+
+  Morehouse then describes how these two strategies are combined in
+  LND's new sweeper system: "[HTLC][topic htlc] claims with matching
+  deadlines [are aggregated] into a single [batched transaction][topic
```

Comment:
> [are aggregated]

Is this a dead link?

Reply:
I guess these are externally inserted words into quote.

Reply:
I see. thanks!